diem-move/diem-framework/core/transactional-tests/0L/diem_account/transfer_community_wallet_script.move

```diff
@@ -14,8 +14,6 @@ script {
 
   fun main(_dr: signer, bob: signer) {
     // Genesis creates 6 validators by default which are already slow wallets,
-    // adding Bob
-    DiemAccount::set_slow(&bob);
```

Comment:
@0xzoz 
Bob is a validator. As such, he should be set to slow.
Why removing this here?

Reply:
Its a test script. Bob is not a validator in this instance

https://github.com/OLSF/libra/blob/36439a7064b3f0bc3b5cad0641aac17e88884224/diem-move/diem-framework/core/transactional-tests/0L/diem_account/transfer_community_wallet_script.move#L3